### ISSUE TITLE
Add official documentation of NOS codes to CourtListener

### ISSUE THREAD

**Issue by mlissner** [freelawproject/courtlistener#857]
They're documented here: http://www.uscourts.gov/sites/default/files/js_044_code_descriptions.pdf

We could probably link to this under the NOS search boxes and in the advanced search docs.

**Comment by davidxia** [comment]
Plan on picking this up after #541.

**Comment by mlissner** [comment]
Great! I'll be nice just to have a simple link to this in the right places! Open question is whether we should host this file ourselves, or trust the AO to not move it around...I'd say we host it ourselves at the risk that we don't have the latest version. That's a better place to be than a broken link we don't know about until somebody tells us.

**Comment by johnhawkinson** [comment]
These NOS codes change with some frequency, as partially evidenced by the recent rev date of that document. I believe there were CMECF release notes about NOS code changes in the past year, and  the [Judicial Conference minutes from March of this year](http://www.uscourts.gov/sites/default/files/18-03_proceedings_0.pdf) approved another change?

**Therefore:** Link, with an ancillary link to our local mirror, and please add to the list of external URLs to automatically validate for broken links. Does CL run a periodic sitewide linkchecker that would obviate the need for that list? (If no such list, then open an Issue?).

**Comment by johnhawkinson** [comment]
> We could probably link to this under the NOS search boxes and in the advanced search docs.

Linking by itself doesn't really seem great, at least aspirationally. We should parse and present appropriate HTML. People should not have to download a PDF to figure out what to put in a search box.

**Comment by mlissner** [comment]
I'm not down with doing two links. That'll just confuse most people, but how about we host it, and in your issue about checking broken links (which we don't do currently), you can mention this issue and the goal of replacing our link with one to the uscourts? 

Agree re converting to HTML. But I sure don't want to do it. Much better would be a auto-complete or something, but I'd be happy just to have a reference in the near term. Probably other higher priorities (both in UX and elsewhere).

**Comment by mlissner** [comment]
Good point about scraping this. Let's start with the link though, since @davidxia is working on that. Progress not perfection for this one, though I'd like to do what you're proposing eventually. We have so much low hanging fruit that'll make biggish UX gains. I don't want to bog down on something like this until more of that is wiped out.
